app/scripts/controller-init/token-balances-controller-init.ts

```diff
@@ -13,6 +13,12 @@ export const TokenBalancesControllerInit: ControllerInitFunction<
   const { useMultiAccountBalanceChecker } = initMessenger.call(
     'PreferencesController:getState',
   );
+  const state = initMessenger.call('RemoteFeatureFlagController:getState');
+
+  const FEATURE_FLAG_NAME = 'assetsAccountApiBalances';
+
+  const featureFlagForAccountsApiBalances =
+    state?.remoteFeatureFlags?.[FEATURE_FLAG_NAME] ?? [];
```

Comment:
Since this data is `any` should we do some simple data parsing?
```ts
const getAccountAPIBalancesFF = () => {
  const balances = state?.remoteFeatureFlags?.[FEATURE_FLAG_NAME] ?? [];
  if (Array.isArray(balances) && balances.every(b => typeof b === 'string')) {
    return balances;
  }
  return []
}

```

Reply:
meh this is a nit  - non blocking